low-level/sensors/impl/generic/node_hw.py

```diff
@@ -480,6 +481,54 @@ def run(self):
             logger.warning(f"{self.SENSOR_NAME} Node hw monitoring disabled")
             self.shutdown()
 
+    def _check_missmatch_resource_status(self):
+        for sensor_id in self.faulty_resources:
+            if self.faulty_resources[sensor_id]['resource'] in ['fan', 'disk', 'temperature', 'voltage']:
```

Comment:
We need to include 'Power Supply' fru as well here.
Below would be suffice. Please check.
`if self.faulty_resources[sensor_id]['resource'].title() in self.fru_types.keys():`